addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -368,10 +368,16 @@ export class HtmlField extends Component {
      * @param {Object} position
      */
     positionDynamicPlaceholder(popover, position) {
-        let topPosition = this.wysiwygRangePosition.top;
+        // make sure the popover won't be out(below) of the page
+        let enoughSpaceBelow = window.innerHeight - popover.clientHeight - this.wysiwygRangePosition.top;
+        let topPosition = (enoughSpaceBelow > 0) ? this.wysiwygRangePosition.top : this.wysiwygRangePosition.top + enoughSpaceBelow;
```

Comment:
We never change the value of `enoughSpaceBelow` and `enoughSpaceRight`, they should therefore be `const` instead of `let`.

Reply:
Done, thanksss :pray: